codegen/tests/simple/app.go

```diff
@@ -0,0 +1,37 @@
+// Code generated by sysl DO NOT EDIT.
+package simple
+
+import (
+	"github.com/anz-bank/sysl-go/codegen/tests/deps"
+	"github.com/anz-bank/sysl-go/codegen/tests/downstream"
+	"github.com/anz-bank/sysl-go/config"
+	"github.com/anz-bank/sysl-go/core"
+	"github.com/anz-bank/sysl-go/handlerinitialiser"
+)
+
+// HandlerInit for Simple
+type HandlerInit struct {
+	enabledHandlers     []handlerinitialiser.HandlerInitialiser
+	enabledGrpcHandlers []handlerinitialiser.GrpcHandlerInitialiser
+}
+
+// InitialiseHandler ...
+func InitialiseHandler(coreCfg *config.DefaultConfig, serviceInterface ServiceInterface, serviceCallback core.RestGenCallback) (*HandlerInit, error) {
+	var err error = nil
```

Comment:
This value isn't used except to return nil at the end of the function. Is this deliberate?

Reply:
Yes, workaround to make codegen work.